src/components/charts/pie-chart.tsx

```diff
@@ -53,17 +54,8 @@ export class PieChart extends BaseComponent<IProps, IState> {
             return percentage;
           },
           color(ctx: any) {
-            let labelColor = "";
-            switch (ctx.dataset.backgroundColor[ctx.dataIndex]) {
-              case "#bf8f53":
-              case "#dfc39d":
-              case "#FEFF59":
-                labelColor = "#000000";
-                break;
-              default:
-                labelColor = "#ffffff";
-            }
-            return labelColor;
+            const backgroundColor = ctx.dataset.backgroundColor[ctx.dataIndex];
+            return colorIsLight(backgroundColor) ? "#000" : "#fff";
```

Comment:
We are probably going to need to have a general discussion about how to make this text visible when we have small, adjacent pie slices.  Might not need to be addressed now, but you can see in the image below that we're eventually going to need to come up with a solution to make the text easier to read in some cases:
![image](https://user-images.githubusercontent.com/5126913/80750327-b9a11000-8adc-11ea-93d6-b5422c2c8275.png)

Reply:
Agreed.